<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user</class>
 <widget class="QWidget" name="user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>参选对象</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>票数</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>排名</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>百分比</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>441</width>
     <height>361</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>441</width>
     <height>361</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>441</width>
     <height>361</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>440</width>
      <height>900</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>440</width>
      <height>900</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>89</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>当前活动：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="activityName">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="signUpbtn">
      <property name="text">
       <string>报名</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="searchbtn">
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>搜索：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchinput">
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>11</number>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>请输入完整电话号码查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="5">
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4" rowspan="2" colspan="2">
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="voteOrregister">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="remaindDay">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="passDay">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="changecodebtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>250</y>
     <width>83</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>改密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changenamebtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>330</y>
     <width>83</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>改用户名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backmainbtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>420</y>
     <width>83</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
